src/compositor/lipstickcompositor.cpp

```diff
@@ -732,3 +737,35 @@ void LipstickCompositor::readContent()
 {
     m_recorder->recordFrame(this);
 }
+
+void LipstickCompositor::timerEvent(QTimerEvent *e)
+{
+    if (isVisible()) {
+        stopRendering();
```

Comment:
Yes. Previously mce grabbed touch as soon as pending touch ended - so that ui would see the touch release event. Now it grabs immediately when display power off starts - assumption being that ui side will self cancel unfinished swipes / etc when display gets powered off.